dll/cpl/desk/theme.c

```diff
@@ -997,8 +997,6 @@ DrawThemePreview(IN HDC hdcMem, IN PCOLOR_SCHEME scheme, IN PTHEME_SELECTION pSe
     FillRect(hdcMem, prcWindow, hbrBack);
     DeleteObject(hbrBack);
 
-    InflateRect(prcWindow, -10, -10);
```

Comment:
Does this also fix how the preview is shown, _**when testing our desk.cpl on Windows XP or 2003**_ ?

Reply:
I wonder whether it's possible to test our desk.cpl+uxtheme.dll on 2003 👀 Just to make sure it's consistent.

Reply:
OK @Splitwirez rechecking this change, so what would the change modify in the screenshot above? the distance between the preview "control" and the previewed sample windows drawn inside?